miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/form/QcTableTag.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2002-2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.bbsrc.tgac.miso.webapp.util.form;
+
+import java.util.stream.Collectors;
+
+import org.springframework.web.servlet.tags.RequestContextAwareTag;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import uk.ac.bbsrc.tgac.miso.core.data.QualityControllable;
+import uk.ac.bbsrc.tgac.miso.dto.Dtos;
+
+@SuppressWarnings("serial")
+public class QcTableTag extends RequestContextAwareTag {
+  private static long id;
+  private Object item;
+
+  @Override
+  protected int doStartTagInternal() throws Exception {
+
+    QualityControllable<?> item = (QualityControllable<?>) this.item;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "item" which hides the field declared at line 31. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)